### DISCUSSION TITLE
Requesting for a small change in package.json

### DISCUSSION THREAD

**Discussion by itzkushagra** [JhohannesK/TipoHiZ#105] [General]
adding this line:
      "install-all": "cross-env npm install && cd backend && npm install && cd ../frontend && npm install"
in the script in order to install all the dependencies as the other mentioned commands aren't installing all of the required dependencies. Is it okay if I do that?

**Comment by itzkushagra** [comment]
@JhohannesK  awaiting your response

**Comment by JhohannesK** [comment]
oops, yeah, right

forgot to change this when we dissolved the frontend and backend folders.

**Comment by JhohannesK** [comment]
since you're working on something, 
lemme just change this real quick and push so everyone else get's the update as soon as possible.
unless you're close to done or now beginning

**Reply by itzkushagra** [reply]
I was about create the pr request for 1st issue (I have almost completed the 2nd issue too), thats why I asked.

**Reply by JhohannesK** [reply]
if we would be getting a PR soon from then you can add it, if not then create an issue so we pick that up quickly

**Comment by itzkushagra** [comment]
Sure I am creating the PR then
